lib/luadata.h

```diff
@@ -6,16 +6,23 @@
 #ifndef luadata_h
 #define luadata_h
 
+#include <lunatik.h>
+
 LUNATIK_LIB(data);
 
+typedef enum {
+	LUADATA_CONST = 1,
```

Comment:
my point on using a flag was to handle every "bool" with it.. e.g., `LUADATA_SLEEP` instead of having a separate field in the struct for it.. (and anything else we need in the future)